test/sharness/t0126-twonode-quic.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+test_description="Test two ipfs nodes transferring a file via QUIC"
```

Comment:
Let's just add a case to the existing twonode test. It's designed for multiple variants (just run `run_advanced_test` again after configuring the nodes to use QUIC).

Reply:
I was worried about the test output. By putting it in one file, it's hard to see which test exactly is failing. I added some more logging, so that shouldn't be a problem anymore.